content/shmem_calloc.tex

```diff
@@ -0,0 +1,40 @@
+\apisummary{
+  Allocate a zeroed block of symmetric memory.
+}
+
+\begin{apidefinition}
+
+\begin{Csynopsis}
+void *shmem_calloc(size_t nmemb, size_t size);
+\end{Csynopsis}
+
+\begin{apiarguments}
+  \apiargument{IN}{nmemb}{The number of elements to allocate.}
```

Comment:
Because `size` is already used for the per-element size. The C standard uses `nmemb` for `calloc()`; POSIX uses `nelem`.